web/crypto_utils.py

```
@@ -0,0 +1,149 @@
+        return value
+
+    def to_python(self, value):
+        if isinstance(value, str) and value.startswith("gAAAAA"):  # Likely encrypted
+            value = decrypt_value(value)
+        return super().to_python(value)
+
+
+class EncryptedEmailField(models.EmailField):
+    """EmailField that encrypts its contents before saving to the database."""
+
+    def get_prep_value(self, value):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_prep_value`

(ANN201)